### PR TITLE
HIHCCDEV-1746: Fix JS error in TOC view of SOLGM

### PR DESCRIPTION
**Pull request by EEEEEBanini**
Updated the index.js to not check for iframe embedding.

### CODE REVIEW

File: lib/js/index.js [hunk 1]
```diff
@@ -6,13 +6,15 @@ var _cernerSmartEmbeddableLib2 = _interopRequireDefault(_cernerSmartEmbeddableLi
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
-if (window.external && typeof window.external.DiscernObjectFactory !== 'undefined' && window.self !== window.top) {
+window.CernerSmartEmbeddableLib = window.CernerSmartEmbeddableLib || {}; /* globals window */
+
+// eslint-disable-next-line no-unused-vars
+
+
```
Comment (cerner-area51) [inline lib/js/index.js:13]: nit: blank lines

File: lib/js/index.js [hunk 2]
```diff
@@ -6,13 +6,15 @@ var _cernerSmartEmbeddableLib2 = _interopRequireDefault(_cernerSmartEmbeddableLi
...
   _cernerSmartEmbeddableLib2.default.listenForCustomFrameHeight();
 
-  window.CernerSmartEmbeddableLib = window.CernerSmartEmbeddableLib || {};
   window.CernerSmartEmbeddableLib.calcFrameHeight = _cernerSmartEmbeddableLib2.default.calcFrameHeight;
   window.CernerSmartEmbeddableLib.setFrameHeight = _cernerSmartEmbeddableLib2.default.setFrameHeight;
-} /* globals window */
-
-// eslint-disable-next-line no-unused-vars
\ No newline at end of file
+}
```
Comment (cerner-area51) [inline lib/js/index.js:20]: nit: new line

**Review by cerner-area51 — APPROVED** [review]

**Review by shruti-gopinath — APPROVED** [review]

File: lib/js/index.js (hunk 1 above)
Comment (EEEEEBanini) [reply lib/js/index.js:13]: This file's autogenerated by npm

File: lib/js/index.js (hunk 2 above)
Comment (EEEEEBanini) [reply lib/js/index.js:20]: This file's autogenerated by npm

**Review by Roshni27 — APPROVED** [review]

**Pull request merged**